DictionaryTypes.txt

```diff
@@ -295,7 +297,14 @@ enum(coil) "Sensor coil type"
     kit_grad	       6001  "KIT gradiometer"
     kit_ref_mag	       6002  "KIT reference magnetometer"
     baby_grad	       7001  "Baby-SQUID gradiometer"
-    baby_mag	       7002  "Baby-SQUID magnetometer"
+    baby_mag_inner     7002  "BabyMEG inner layer magnetometer"
+    baby_mag_outer	   7003  "BabyMEG outer layer magnetometer"
+    baby_mag_ref	   7004  "BabyMEG reference layer magnetometer"
+    artemis123_grad    7501  "Artemis123 gradiometer"
+    artemis123_ref_mag 7502  "Artemis123 reference magnetometer"
+    artemis123_ref_grad 7503  "Artemis123 reference gradiometer"
+    kriss_grad         8001  "KRISS gradiometer"
```

Comment:
@larsoner Can you pls double check if you switched the value of kriss_grad with sample_tms_planar? The mne-python coil_def.dat [https://github.com/mne-tools/mne-python/blob/master/mne/data/coil_def.dat](url) suggest otherwise.

Reply:
Over in our [constants file](https://github.com/mne-tools/mne-python/blob/master/mne/io/constants.py#L843) it's 8001 but in the `coil_def` I see it's 9001, so looks like our constants file has a bug! I've pushed a commit to fix it here and will make an MNE PR shortly.